regression/sql/denorm_prepared_statements.sql

```diff
@@ -0,0 +1,19 @@
+CREATE EXTENSION pg_stat_monitor;
+Set pg_stat_monitor.pgsm_normalized_query='off';
+
+CREATE TABLE t1 (a TEXT, b TEXT, c TEXT);
+
+SELECT pg_stat_monitor_reset();
+
+PREPARE prepstmt(TEXT, TEXT, TEXT) AS INSERT INTO t1(a, b, c) VALUES($1, $2, $3);
+
+EXECUTE prepstmt('A', 'B', 'C');
+SELECT query, calls FROM pg_stat_monitor ORDER BY query COLLATE "C";
```

Comment:
Just to make it clear that we are checking exact same data:
```suggestion
SELECT SUBSTRING(query, 0, 128), calls FROM pg_stat_monitor ORDER BY query COLLATE "C";
```

Reply:
Good, fixup on https://github.com/percona/pg_stat_monitor/pull/481/commits/20dd640b1e935fb35daaf65148e9e6104617cb3c